algorithms/bellman_ford.py

```diff
@@ -0,0 +1,133 @@
+import numpy as np
+import graphviz
+
+## Defining classes for vertex and Graph
+
+class vertex :
+    def __init__(self,distance=0,pre=0):
+        self.d=distance
+        self.p=pre
+
+        
+class Graph :
+    def __init__(self,vertices=0,edges=0):
+
+        self.vertices=vertices
+        self.edges=edges
+   
+    def single_source(self,j):
+        k=list(self.edges.values())
+        k=np.array(k)
+        k=abs(k)
+        max_edge= k.sum()
+        for i in self.vertices:
+            self.vertices[i].d= max_edge+1
+        self.vertices[j].d=0
+        return None
+            
+    def relax(self,j):
+        if self.vertices[int(j[3])].d > self.vertices[int(j[1])].d +self.edges[j]:
+           self.vertices[int(j[3])].d =  self.vertices[int(j[1])].d +self.edges[j]
+           self.vertices[int(j[3])].p= int(j[1])
+           return None
+             
+        
+
+
+    def bellman_ford(self,j):
+        self.single_source(j)
+        for i in range(len(self.vertices)-1):
+            for i in self.edges:
+                self.relax(i)
+        for i in self.edges:
+            if self.vertices[int(i[3])].d >  self.vertices[int(i[1])].d +self.edges[i]:
+                return False
+        return self.vertices
+
+
+
+
+
+  ## Taking inputs for edges and vertices for graph in order to perform Bellman Ford.
+
+
+    
+
+    
+num_vertices= int(input('Input the number of vertices in the graph: '))
+
+dict_vertices= {}
+
+for i in range(1,num_vertices+1):
+    dict_vertices[i]= vertex()
+ 
+d=input('The name of the vertices is numeric that is from 1 to n where n is the number of vertices. Press any key to continue : ')
```

Comment:
```suggestion
d=input('The name of the vertices is numeric that is from 1 to n where n is the number of vertices.\nPress any key to continue : ')
```
As the line is too long!